playground/negotiator-authenticator.js

```diff
@@ -0,0 +1,172 @@
+// import Negotiator()
+// import Authenticator()
+
+const negotiatorAuthenticator = {};
+const currentUser = 2;
+
+// Helper for success and failed promises
+const asyncHelper = async (promise) => {
+    try {
+        const data = await promise;
+        return [data, undefined];
+    } catch (error) {
+        return [undefined, error];
+    }
+}
+
+// Mock of Negotiator
+const Negotiator = {
+    getTokenInstances: function async() {
+        return [
+            { id: 42, token: { ticketClass: "VIP" }, ownerAddress: null },
```

Comment:
Oh, I see. You have the `id` outside of token object. Well actually it is inside because it needs to be signed (unlike `ownerAddress` which needn't to be signed. But it's up to you if you want to create an artificial `id` which is not the actual `ticetID`, just as long as the web developer know that they can't use this artificial `id` when communicating with the blockchain.

Reply:
I'll bring all the attributes into the token as per example below (and remove the 'id' from the object):
{ token: { ticketId: 42, ticketClass: "VIP", conferenceId: 1 }, ownerAddress: null },